src/components/utils/include/utils/qdb_wrapper/sql_database.h

```diff
@@ -91,6 +91,19 @@ class SQLDatabase {
    */
   bool Backup();
 
+	/**
```

Comment:
@BSolonenko indents...

Reply:
@AByzhynar fixed in https://github.com/Ford-Luxoft/sdl_core/commit/a3a73a446d1a8fe30f7b98dde719b35719b6d731